src/containers/Bookform.js

```diff
@@ -8,16 +23,56 @@ const BookForm = () => {
       {category}
     </option>
   ));
+  return options;
+};
+
+const BookForm = (props) => {
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+  const options = categories();
+
+  const handleChange = (e) => {
+    if (e.target.id === 'title') {
+      setTitle(e.target.value);
+    } else {
+      setCategory(e.target.value);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const book = {
+      id: createUniqueId(),
+      title,
+      category,
+    };
+
+    const { createbook } = props;
+    createbook(book);
+
+    setTitle('');
+    setCategory('');
+  };
```

Comment:
- [ ] <b>REQUIRED:</b> this `handleSubmit` is not working as it is supposed to. it should be able to create a new book item, store it in the store and display it on the screen. but it is not working. kindly fix it.